lib/money/money_parser.rb

```diff
@@ -22,7 +22,7 @@ class MoneyFormatError < ArgumentError; end
     (?:
       (?:\d+)
       (?:[#{ESCAPED_NON_DOT_MARKS}]\d{3})+
-      (?:\.\d+)?
+      (?:\.\d{2,})?
```

Comment:
we only support 2 decimals? Example currencies that have 3 decimals: `BHD`, `IQD`,...

Reply:
`{2,}` is 2+

Reply:
how bad can I be reading a regex that I don't write. damn me. you're right